### DISCUSSION TITLE
RBAC scopes cognito:groups

### DISCUSSION THREAD

**Discussion by andrewhearin** [argoproj/argo-cd#11099] [Q&A]
I was just wondering if there's a reason the [documentation](https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-rbac-cm.yaml#L27-L29) for the rbac config map manifest has "cognito:groups" as a scope default.  

```sh
# scopes controls which OIDC scopes to examine during rbac enforcement (in addition to `sub` scope).
# If omitted, defaults to: '[groups]'. The scope value can be a string, or a list of strings.
scopes: '[cognito:groups, email]'
```
1. Does this refer to AWS Cognito? 
3. If so, should there be a comment that makes this more clear?  (instead of making assumptions on the end user's provider)

thanks!

**Comment by puru-yanamala** [comment]
Could somebody answer to this query here please ?

**Comment by LuWang1983** [comment]
I have the exact same question and request but it seems like it does associate with AWS cognito. See [this documentation](https://gist.github.com/vavdoshka/f180bc49886728d9f051d9c4db045f9e).
